sdk-extensions/autoconfigure/src/testFullConfig/java/io/opentelemetry/sdk/autoconfigure/FullConfigTest.java

```diff
@@ -197,7 +197,7 @@ void configures() throws Exception {
               .contains(
                   KeyValue.newBuilder()
                       .setKey("configured")
-                      .setValue(AnyValue.newBuilder().setStringValue("true").build())
+                      .setValue(AnyValue.newBuilder().setBoolValue(true).build())
```

Comment:
interesting. is this a behavioral change somehow?

Reply:
Nah I just made the test code consistent between trace and metrics for the test SPI implementations

Reply:
cool. yeah, I think I figured that out eventually working through here. 👍🏽